docs/en/use-cases/observability/schema-design.md

```diff
@@ -1311,7 +1313,7 @@ WHERE TraceId != ''
 GROUP BY TraceId
 ```
 
-The view effectively ensures the table `otel_traces_trace_id_ts` has the minimum and maximum Timestamp for the trace. This table, ordered by `TraceId`, allows these timestamps to be retrieved efficiently. These timestamp ranges can, in turn, be used when querying the main `otel_traces` table. More specifically, when retrieving a trace by its id, Grafana uses the following query:
+The view effectively ensures the table `otel_traces_trace_id_ts` has the minimum and maximum timestamp for the trace. This table, ordered by `TraceId`, allows these timestamps to be retrieved efficiently. These timestamp ranges can, in turn, be used when querying the main `otel_traces` table. More specifically, when retrieving a trace by its id, Grafana uses the following query:
```

Comment:
This is referring to max and min timestamps in general I think, not some specific column name called `Timestamp`?